projects/media-viewer/src/lib/icp/icp-session.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
```

Comment:
maybe this file could be renamed to include api, as that seems to be consistent with the rest of the app